build.sbt

```diff
@@ -155,12 +157,15 @@ lazy val core = crossProject(JSPlatform, JVMPlatform)
   .settings(moduleName := "algebra")
   .settings(
     mimaPreviousArtifacts := Set("org.typelevel" %% "algebra" % binaryCompatibleVersion),
-    mimaBinaryIssueFilters ++= ignoredABIProblems
```

Comment:
Why did you remove this?

Reply:
Removing unneeded parens shouldn't change mima. The JVM binary generated should be the same.

Reply:
Actually it wasn't removed it was moved down,the diff doesn't make it very clear